src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
By "default" you mean `IECore::GeometricData::None`? The ones created on OSLShaders will have the "correct" interpretation set. I just tried creating a oslBuildColor node and when I query the interpretation I get `IECore.GeometricData.Interpretation.Color` while not getting that in the serialisation. I am happy to remove it from the conditional if you're ok with that showing up in the serialisation :)